core-web/libs/template-builder/src/lib/components/template-builder/components/template-builder-box/template-builder-box.component.html

```diff
@@ -9,8 +9,8 @@
     [style]="{ width: '542px' }"
     [draggable]="false"
     [resizable]="false"
-    [blockScroll]="true"
     [modal]="true"
+    [dismissableMask]="true"
```

Comment:
Why?

Reply:
@fmontes This is about the thing we talked about on Friday being able to dismiss the pop-up when you click outside.

Reply:
True, ignore this why :D